check-instance.yml

```diff
@@ -24,49 +24,5 @@
   vars:
     allow_install_on_vm: false
```

Comment:
This was added to roles/check-instance/defaults/main.yml;  is it possible to remove here?

Reply:
Will move.